components/Steps/index.jsx

```diff
@@ -610,59 +353,30 @@ export const inputLabels = {
           <ul>
             <li>
               My business no longer occupies the premises stated in this
-              application form
+              application form.
             </li>
             <li>
               My business ceases trading permanently, or goes into
               administration, becomes insolvent, or is in receipt of a striking
-              off notice
-            </li>
-            <li>
-              My business no longer meets any other grant eligibility criteria
+              off notice.
             </li>
           </ul>
         </>
       ),
       validation: { required: true },
     },
-    businessNotExceed: {
-      label: (
-        <>
-          <p>
-            The Additional Restrictions Grant counts towards the total de
-            minimis state aid you are permitted to receive over a 3 year period
-            which is €200,000. If you have reached the de minimis threshold, you
-            may still be eligible for funding under the{' '}
-            <a
-              href="https://ec.europa.eu/competition/state_aid/what_is_new/covid_19.html"
-              target="_blank"
-              rel="noopener"
-            >
-              COVID-19 Temporary Framework
-            </a>
-            . The limit for this framework is €800,000.
-          </p>
-          <p>
-            I confirm that, including receipt of this grant, the business will
-            not exceed the relevant State Aid threshold.
-          </p>
-        </>
-      ),
-      validation: { required: true },
-    },
-    businessNotUndertaking: {
+    businessNotExceedPermittedAllowance: {
       label: (
         <>
-          I confirm that my business is not an{' '}
+          I confirm that I have read the{' '}
           <a
-            href="https://www.gov.uk/guidance/innovation-apply-for-a-funding-award#undertakings-in-difficulty--eu-definition"
+            href="https://drive.google.com/file/d/1--cmWYpNzr0SX6dkkbiXDocXIoEuKuQc/view"
```

Comment:
This doesn't appear to be available publicly?

Reply:
I guess we could embed it. Or make public. I've noted this so will pick up with Neil, alongside acquiring the other missing link.